packages/x-date-pickers-pro/src/DateRangePicker/dateRangePickerToolbarClasses.ts

```diff
@@ -0,0 +1,19 @@
+import { generateUtilityClass, generateUtilityClasses } from '@mui/material';
+
+export interface DateRangePickerToolbarClasses {
+  /** Styles applied to the root element. */
+  root: string;
+  /** Styles applied to the container element. */
+  container: string;
+}
+
+export type DateRangePickerToolbarClassKey = keyof DateRangePickerToolbarClasses;
+
+export function getDateRangePickerToolbarUtilityClass(slot: string) {
+  return generateUtilityClass('MuiDateRangePickerToolbar', slot);
+}
+
+export const dateRangePickerToolbarClasses: DateRangePickerToolbarClasses = generateUtilityClasses(
+  'MuiDateRangePickerToolbar',
+  ['root', 'container'],
+);
```

Comment:
I do not see where its used, and it does not seems to be exported for developpers

Reply:
You mean that we should export these to be used for nested style overriding? 🤔

Reply:
Did I cover what you saw missing with [this commit](https://github.com/mui/mui-x/pull/6096/commits/a2c4a5d94908f3d332075bf4eb92bebee320413b)?

Reply:
Yes, it seems that other classes are exported like that 🤔

I nearly neither use those customization features, so I'm not aware of all the subtilities. I was just curious to know why those classes were added